validator/node/node.go

```diff
@@ -456,7 +457,8 @@ func web3SignerConfig(cliCtx *cli.Context) (*remoteweb3signer.SetupConfig, error
 		if cliCtx.IsSet(flags.WalletPasswordFileFlag.Name) {
 			log.Warnf("%s was provided while using web3signer and will be ignored", flags.WalletPasswordFileFlag.Name)
 		}
-		if cliCtx.IsSet(flags.Web3SignerPublicValidatorKeysFlag.Name) {
+
+		if cliCtx.IsSet(flags.Web3SignerPublicValidatorKeysFlag.Name) || len(cliCtx.StringSlice(flags.Web3SignerPublicValidatorKeysFlag.Name)) > 0 {
 			publicKeysSlice := cliCtx.StringSlice(flags.Web3SignerPublicValidatorKeysFlag.Name)
```

Comment:
```suggestion
		if publicKeysSlice := cliCtx.StringSlice(flags.Web3SignerPublicValidatorKeysFlag.Name); len(publicKeysSlice) > 0
```

This might be a better approach. We don't really care if the flag is set, right? we really only care if publicKeysSlice has items in the slice.